.github/workflows/ci.yml

```diff
@@ -188,3 +188,26 @@ jobs:
         path: /home/runner/work/avocado/avocado/dist/
         retention-days: 1
     - run: echo "🥑 This job's status is ${{ job.status }}."
+
+  experimental-checks:
+    name: Experimental checks
+    runs-on: ubuntu-20.04
+    steps:
+      - run: echo "Job triggered by a ${{ github.event_name }} event on branch is ${{ github.ref }} in repository is ${{ github.repository }}, runner on ${{ runner.os }}"
+      - name: Check out repository code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Run Codespell Check
+        run: make -f Makefile.gh codespell
```

Comment:
Since the makefile is inside the same directory, what about avoiding the .gh extension so we don' t need the -f?

Are you planning to have multiple Makefiles inside this directory?

Reply:
`Makefile.gh` is in the root directory of avocado, we can't remove the .gh extension because we already have a Makefile there.
 I'm planning to have multiple Makefile in the root directory. A casual contributor will only need to care for the main `Makefile` and we'll have the extra Makefiles for development purposes. I considered adding them in a subdirectory, but it was add extra complexity in the rules (they need to be run from the root directory) for little gain.

Reply:
My mistake I tough this was inside this directory. Lets go this way, but I think having multiple Makefiles inside the root might be confusing.